data/mods/Magiclysm/Spells/attunements/Crusader.json

```
@@ -52,21 +52,220 @@
     "type": "SPELL",
     "name": "Holy Shield",
     "description": "Crusaders have the ability to summon a magically enhanced shield to their arm to protect them in armed combat.",
-    "valid_targets": [ "none" ],
+    "valid_targets": [ "self" ],
     "flags": [ "CONCENTRATE", "NO_LEGS", "MUST_HAVE_CLASS_TO_LEARN" ],
     "effect": "spawn_item",
     "shape": "blast",
     "effect_str": "holy_shield",
     "min_damage": 1,
     "max_damage": 1,
-    "min_duration": 0,
-    "max_duration": 3240000,
-    "duration_increment": 92572,
+    "min_duration": 360000,
+    "max_duration": 12960000,
+    "duration_increment": 360000,
     "spell_class": "CRUSADER",
-    "difficulty": 1,
+    "difficulty": 4,
     "max_level": 35,
     "base_casting_time": 1500,
     "base_energy_cost": 500,
     "energy_source": "MANA"
+  },
+  {
+    "id": "judgement_sword",
+    "type": "SPELL",
+    "name": "Sword of Judgement",
+    "description": "This blade of light will cut through any evil it makes contact with!  You can change his shape into dagger, great sword, spear or mace.",
```

```suggestion
    "description": "This blade of light will cut through any evil it makes contact with!  You can change its shape into a dagger, great sword, spear or mace.",
```
Unless the sword is sentient, this doesn't make sense 😛